### ISSUE TITLE
Nautilus display problem in Ubuntu 22.04.2 LTS

### ISSUE THREAD

**Issue by See-Night** [vinceliuice/WhiteSur-gtk-theme#888]
# System

* OS: Ubuntu 22.04 LTS
* DE: Gnome 42.5
* Kernel: 5.19.0-45-generic

# Problem

After installing WhiteSur-gtk-theme, nautilus had a display problem: 
![录屏 2023年06月17日 17时48分52秒.webm](https://github.com/vinceliuice/WhiteSur-gtk-theme/assets/45226780/b3d058b6-3c85-422e-8892-74fb0b301760)

**Comment by fr3nchalp4a** [comment]
oh, this smells like graphical bugs with Gnome or Wayland, not a theme issue.

**Comment by fr3nchalp4a** [comment]
Are you on X11 or Wayland ?

**Comment by vinceliuice** [comment]
这个问题解决不了，建议用 solid 版本，只要有透明度在 n 卡 nautilus 上就有这个问题

**Comment by See-Night** [comment]
> Are you on X11 or Wayland ?

I'm on X11, and I still had this problem when trying to boot with a core graphics card. But if I use glassy, everything is fine.

![image](https://github.com/vinceliuice/WhiteSur-gtk-theme/assets/45226780/24c94790-7ade-45fc-a46c-ad96b46fb2a9)

**Comment by fr3nchalp4a** [comment]
And with Wayland, same bug ?
Basic infos about your hardware could help :
- iGPU / dGPU (integrated/dedicated graphics card)
- Which version of X11 and Wayland, etc...

Maybe `looking glass` could help ?

**Comment by shun2wang** [comment]
Yes, this indeed a problem while using Nvidia GPU  driver with nautilus only.

**Comment by akaJuliaan** [comment]
Are there any news on how to fix this issue?

**Comment by fr3nchalp4a** [comment]
@See-Night Is it still present on gnome 45 ? I personally don't have this issue as I am on Intel graphics. If it is still present, check with which combinations of nautilus styles and --darker colors the issue exists.

**Comment by vinceliuice** [comment]
@fr3nchalp4a  Nautilus gtk-4.0 version do not have this issue
